Realm/ObjectServerTests/RLMSyncTestCase.mm

```diff
@@ -175,38 +175,74 @@ + (NSString *)primaryKey {
 }
 
 + (NSArray *)requiredProperties {
-    return @[@"booCol",
-             @"cBoolcol",
-             @"intCol",
-             @"doubleCol",
-             @"stringCol",
-             @"binaryCol",
-             @"dateCol",
-             @"longCol",
-             @"decimalCol",
-             @"uuidCol",
-    ];
-}
-
-+ (NSDictionary *)values:(int)i {
+    return @[@"boolCol", @"cBoolcol",
+             @"intCol", @"doubleCol",
+             @"stringCol", @"binaryCol",
+             @"dateCol", @"longCol",
+             @"decimalCol", @"uuidCol", @"objectIdCol"];
+}
+
++ (NSDictionary *)values:(int)idx {
```

Comment:
The parameter here isn't an index. It's just a number with no particular meaning.